### DISCUSSION TITLE
포탈을 타고 특정 월드로 이동하는 기능을 어떻게 만드나요?

### DISCUSSION THREAD

**Discussion by devinhyeok** [naverz/zepeto-studio-kor#258] [5. Q&A]
두개의 월드를 만든 다음 포탈로 이어주고 싶습니다. 어떻게 해야하나요?

**Comment by cheolhui-lee** [comment]
안녕하세요.
현재는 다른 월드로 이동 하는 기능을 지원하지 않고 있습니다.
대신 유니티 씬 전환을 이용해보시는 것이 어떨까요?

**Reply by devinhyeok** [reply]
![image](https://user-images.githubusercontent.com/94621970/150981999-991db432-ccf8-484f-87b6-53c06733a008.png)
![image](https://user-images.githubusercontent.com/94621970/150982022-8c8238e1-1455-45fe-b110-b37aad4c41ed.png)
![image](https://user-images.githubusercontent.com/94621970/150982044-f80dd9b6-0900-4b93-8008-d270864d8b0c.png)
![image](https://user-images.githubusercontent.com/94621970/150982063-b5305cc1-6aee-464a-97f4-d3da2f7ca05e.png)

제페토 공식맵의 클레스룸 1,2 맵에 포탈을 통해 마이하우스로 이동 가능하던데
다른월드로 이동하는건 왜 안되나요?
기술적으로 다른가요? 혹은 공식맵은 회사에서 만든거라 별도의 작업을 한것인가요?

**Reply by x4cf** [reply]
유니티 일반과 프로, 넥슨의 메이플스토리와 Mode처럼 당연히 제페토도 일반 공개버전의 API와 내부에서 사용하는 API는 다를 거라고 생각합니다.
